tests/unittests/distros/test_netconfig.py

```diff
@@ -381,7 +381,7 @@ def test_apply_network_config_freebsd_ifrename(self, ifaces_mac):
         rc_conf_expected = """\
 ifconfig_vtnet0_name=eth0
 defaultrouter=192.168.1.254
-ifconfig_eth0='192.168.1.5 netmask 255.255.255.0'
+ifconfig_eth0='inet 192.168.1.5 netmask 255.255.255.0'
```

Comment:
if anything in this file test file tests IPv6, i'd add bsd tests too

Reply:
I don't run any BSD other than FreeBSD. The changes to bsd.py where needed to get IPv6 working for FreeBSD, but it shouldn't cause any regression for others.

Reply:
that's not what I meant: I meant: if in this test_netconfig.py there are places that test IPv6, we should add tests for (Free)BSD there too.

Reply:
Ah, I see what you mean. I'll see what I can do

Reply:
https://github.com/canonical/cloud-init/pull/1839/commits/d59f7c8e6942437b8838222ae0418009f4e18d6b